### PR TITLE
Hostile mobs will now aggro if hit by offscreen projectiles

### PR DESCRIPTION
**Pull request by ChangelingRain**
Instead of just walking towards the area they were shot from.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/hostile.dm [hunk 1]
```diff
@@ -75,8 +75,11 @@
 
 /mob/living/simple_animal/hostile/bullet_act(obj/item/projectile/P)
 	if(!target && AIStatus != AI_OFF && !client)
+		if(P.firer && get_dist(src, P.firer) <= aggro_vision_range)
+			var/list/shooter = list(P.firer)
+			FindTarget(shooter, 1)
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/hostile/hostile.dm]: ```FindTarget(list(P.firer), 1)```

**Comment by Repukan** [comment]
I don't mean to be a dick, but would you agree it would be a good idea, in terms of perspective, if you attempt to fight megafauna before you buff/nerf it so you feel how it handles?

**Comment by ChangelingRain** [comment]
If you are abusing offscreen projectiles to fight megafauna you deserve thisssssss

**Comment by Repukan** [comment]
No, i mean in general. I don't cheese megafauna because fighting it with protos or supers is more fun. But lately (more noticeably, the intended colossus buff/drake nerf) seem misplaced or really not the direction you would do changes if you want to make them stronger/weaker.

**Pull request merged**